projects/extension/src/background/ConnectionManager.ts

```diff
@@ -15,6 +15,17 @@ import { StateEmitter, State } from "./types"
 import { NetworkMainInfo, Network } from "../types"
 import { logger } from "@polkadot/util"
 import { ToExtension } from "@substrate/connect-extension-protocol"
+import westend from "../../public/assets/westend.json"
+import kusama from "../../public/assets/kusama.json"
+import polkadot from "../../public/assets/polkadot.json"
+import rococo from "../../public/assets/rococo.json"
+
+export const relayChains: Map<string, string> = new Map<string, string>([
+  ["polkadot", JSON.stringify(polkadot)],
+  ["kusama", JSON.stringify(kusama)],
+  ["rococo", JSON.stringify(rococo)],
+  ["westend", JSON.stringify(westend)],
+])
```

Comment:
Same remark: that's not great, but let's delay fixing this for after #572

Reply:
Yes, for sure, this is definitely pretty bad... However, this was [here before, I just moved it to the correct file](https://github.com/paritytech/substrate-connect/pull/618/files#diff-4de63d371461c5b51ea9a173eccc4c3b114374baa9388b564a3033e112f58924L18-L26) in order to avoid circular dependencies.